### ISSUE TITLE
Power grid (electrical) enhancements

### ISSUE THREAD

**Issue by MishkaFur** [StrangeLoopGames/EcoSuggestions#1298]
![image](https://user-images.githubusercontent.com/61889138/98673671-98ed2900-234f-11eb-8b30-f77be7906d1b.png)

Should perhaps say how much is from Steam Engine, how much from Combustion, how much from Solar or Wind turbines, can then see where the grid is lacking to be eco friendly/renewable

World layer for where power is would be very useful especially for longer term servers where a larger power grid is being attempted such as the below

![image](https://user-images.githubusercontent.com/61889138/98674012-1add5200-2350-11eb-9568-4b596c98e310.png)

As mentioned in other github's, ie 1256 this could end up being expanded out into payments for power/meters etc etc which would effectively overhaul the electricity system of the game rather than most players just grabbing a power source for themselves only

**Comment by johnkslg** [comment]
Yes! An enhancement that allows buying power is definitely in the cards.
